js/main.js

```diff
@@ -0,0 +1,26 @@
+var $photoUrl = document.querySelector('.photourl');
+var $img = document.querySelector('.placeImage');
+var $form = document.querySelector('.form');
```

Comment:
I don't think this is selecting the right element, I'm getting an error in the console on page load: 
![Screenshot 2023-02-07 at 8 20 48 PM](https://user-images.githubusercontent.com/7886753/217432748-cef1780a-b57b-4bcb-9099-0da3de4aadc4.png)